gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -41,6 +41,14 @@ def files(unit_extension)
       end
     end
 
+    def my_dirs
+      dirs = []
+      (SYSTEM_DIRS + USER_DIRS).each do |dir|
+        dirs << dir if @fs.fileExists?(dir)
+      end
+      dirs
```

Comment:
Yeah, `existing_dirs` sounds good.  It wouldn't make sense for that to be a new constant name though if you were to concatenate them together beforehand.